src/pages/Reviews.js

```diff
@@ -1,42 +1,20 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import axios from 'axios';
 import { useUserContext } from "../context/user_context";
 
-const Reviews = ({ productId }) => {
+const Reviews = ({ productId, reviews, onReviewChange }) => {
   // Fetch current user from context provider, and store it in the `user` variable
   const { currentUser: user } = useUserContext();
   //console.log(user.userId); //user id
 
   const [message, setMessage] = useState('');
-  const [reviews, setReviews] = useState([]);
   const [formData, setFormData] = useState({
-    rating: Number,
```

Comment:
`Number` by itself is a function (see screenshot from dev tools console), which was causing an error in the console about the  input element not liking that. If you wanted to just put a default number in there, you could do `Number()` - but I believe that would enter the number `0` in the input. If you want the default value to just be blank, then an empty string works here `''` (even though I know it seems weird to put an empty string in a number input). 

![Uploading Screenshot 2023-07-10 at 4.12.51 PM.png…]()